Moments/Button.swift

```diff
@@ -1,23 +0,0 @@
-//
-//  Button.swift
-//  Moments
-//
-//  Created by user on 30/10/17.
-//  Copyright © 2017 user. All rights reserved.
-//
-
-import Foundation
-import  UIKit
-
-extension AddMomentsPage{
```

Comment:
why extension filename is "Button.swift"

Reply:
i created the the new moments page instead of add moments page .
To customise the button i created the extension